Add docs for Tempo search (#34526)
grafana/grafana: docs/sources/datasources/tempo.md

## Query traces

You can query and display traces from Tempo via [Explore]({{< relref "../explore/_index.md" >}}).

> You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.

Reviewer: ```suggestion
You can search for traces if you set up the trace to logs setting in the data source configuration page. To find traces to visualize, use the [Loki query editor]({{< relref "loki.md#loki-query-editor" >}}). To get search results, you must have [derived fields]({{< relref "loki.md#derived-fields" >}}) configured, which point to this data source.
```

> You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. In order to have search results you need to have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that points to this data source.

Reviewer 2: ```suggestion
You can search for traces if you set up the trace to logs setting in the data source configuration page. Use [Loki's query editor]({{< relref "loki.md#loki-query-editor" >}}) to find traces to visualize. To have search results, you must have [Derived fields]({{< relref "loki.md#derived-fields" >}}) configured that point to this data source.
```

{{< figure src="/static/img/docs/tempo/query-editor-search.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo query editor showing the search tab" >}}

> To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.

Reviewer: ```suggestion
To query a particular trace, select the **TraceID** query type, and then put the ID into the Trace ID field.
```

> To query a particular trace, select TraceID query type and insert the ID into the Trace ID field.

Reviewer 2: ```suggestion
To query a particular trace, select TraceID query type and then insert the ID into the Trace ID field.
```

> {{< figure src="/src/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}

Reviewer 3: ```suggestion
{{< figure src="/static/img/docs/tempo/query-editor-traceid.png" class="docs-image--no-shadow" caption="Screenshot of the Tempo TraceID query type" >}}
```

Assuming you want the image from https://github.com/grafana/website/pull/4510.

Author: yes thank you, I missed this one
